bin/crew

```diff
@@ -823,7 +823,7 @@ def unpack(meta)
       puts "Unpacking archive using 'unzip', this may take a while..."
       _verbopt = @opt_verbose ? '-v' : '-qq'
       system 'unzip', _verbopt, '-d', @extract_dir, meta[:filename], exception: true
-    when /\.(tar(\.(gz|bz2|xz|lz))?|tgz|tbz|txz)$/i
+    when /\.(tar(\.(gz|bz2|xz|lzma|lz))?|tgz|tbz|txz)$/i
       puts "Unpacking archive using 'tar', this may take a while..."
       system "tar x#{@verbose}f #{meta[:filename]} -C #{@extract_dir}", exception: true
     when /\.tar\.zst$/i
```

Comment:
The original indentation was correct.  Also, bump version in `lib/const.rb`?

Reply:
my IDE must have screwed with the indentation I'll fix that and bump the version

Reply:
[2cad62e](https://github.com/skycocker/chromebrew/pull/7113/commits/2cad62ecf3eb20fd18bf7e49cf66a035bb2e6779)